lstchain/calib/camera/r0.py

```diff
@@ -412,6 +450,33 @@ def _load_calib(self):
                 self.pedestal_value_array[:, :, self.size4drs:self.size4drs + 40] \
                     = pedestal_data[:, :, 0:40] - self.offset
 
+        if not self.calibration_path:
```

Comment:
I think this should fail if no calibration path is give, not just silently return None. raise ValueError or a config error.

Reply:
That was what I though first too, but during the creation of the calibration file itself the LSTR0Corrections is used.

Reply:
I don't understand how this relates to the fact that a function that **loads** a file should do nothing.

Reply:
ah, ok

Reply:
Maybe split this in `load_pedestal` and `load_calibration` then? So each function only does one thing?